x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/external_links.tsx

```diff
@@ -105,22 +105,26 @@ const ObservabilityAppLink: FunctionComponent<Props> = ({ checkpoint }) => {
 };
 
 export const ExternalLinks: FunctionComponent<Props> = ({ checkpoint }) => {
+  const { isInfraPluginAvailable } = useAppContext();
+
   return (
     <EuiFlexGroup>
-      <EuiFlexItem>
-        <EuiPanel>
-          <EuiText size="s">
-            <p>
-              <FormattedMessage
-                id="xpack.upgradeAssistant.overview.observe.observabilityDescription"
-                defaultMessage="Get insight into which deprecated APIs are being used and what applications you need to update."
-              />
-            </p>
-          </EuiText>
-          <EuiSpacer size="m" />
-          <ObservabilityAppLink checkpoint={checkpoint} />
-        </EuiPanel>
-      </EuiFlexItem>
+      {isInfraPluginAvailable && (
```

Comment:
If we implement my suggestion then this would change to something like `useAppContext().plugins.infra`.